tests/utils.py

```diff
@@ -1,25 +1,56 @@
 from pathlib import Path
+from textwrap import dedent
 
-import numpy as np
 import piq  # type: ignore
 import torch
 import torch.nn as nn
 from PIL import Image
 from transformers import T5EncoderModel, T5Tokenizer  # type: ignore
 
+from refiners.conversion.models import dinov2
+from refiners.fluxion.utils import image_to_tensor
+from refiners.foundationals.dinov2 import DINOv2_small
 
-def compare_images(img_1: Image.Image, img_2: Image.Image) -> tuple[int, float]:
-    x1, x2 = (
-        torch.tensor(np.array(x).astype(np.float32)).permute(2, 0, 1).unsqueeze(0) / 255.0 for x in (img_1, img_2)
+dinov2_small_model = DINOv2_small()
+dinov2_small_model.load_from_safetensors(
+    dinov2.small.converted.local_path
+    if dinov2.small.converted.local_path.exists()
+    else dinov2.small.converted.hf_cache_path
+)
```

Comment:
Don't do that (load a model when requiring a very frequently requested helper). Use a function + functools cache.

Reply:
ooops, right my bad